packages/contracts/src/core/permission/PermissionManager.sol

```diff
@@ -332,10 +330,7 @@ contract PermissionManager is Initializable {
     /// @param _where The address of the target contract for which the permission is required.
     /// @param _permissionId The permission identifier required to call the method this modifier is applied to.
     function _auth(address _where, bytes32 _permissionId) private view {
-        if (
-            !isGranted(address(this), msg.sender, _permissionId, msg.data) &&
-            !isGranted(_where, msg.sender, _permissionId, msg.data)
-        ) {
+        if (!isGranted(_where, msg.sender, _permissionId, msg.data)) {
```

Comment:
Interesting..

shouldn't we have `address(this)` ? What made you leave `_where` instead of `address(this)`. Just asking, so we don't miss anything.

Reply:
Yes, I also realized this. It can be simplified even more afterward